expenses/views.py

```diff
@@ -189,6 +190,21 @@ def delete_expense(request, pk):
         messages.success(request, 'Kvittot raderades.')
         return HttpResponseRedirect(reverse('expenses-index'))
 
+@require_POST
+@login_required
+@user_passes_test(lambda u: u.profile.is_admin())
```

Comment:
Prefer adding a new method to [`Profile`](https://github.com/datasektionen/cashflow/blob/4eea3046fb0ca3f3f41317e9939d23386718b274/expenses/models.py#L38) like `may_flag`, which can then defer to e.g. `may_attest or may_pay` for now (unsure what's the intended behavior for who should be able to pay)